pkgs/applications/altcoins/default.nix

```diff
@@ -1,4 +1,4 @@
-{ callPackage, boost155, boost162, boost163, openssl_1_1_0, haskellPackages, darwin, libsForQt5 }:
+{ callPackage, boost155, boost162, openssl_1_1_0, haskellPackages, darwin, libsForQt5 }:
```

Comment:
This build needs both boost155 and boost162?

Reply:
It's a subpackage tree, some altcoins seem to need older Boosts.